common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1093,6 +1093,10 @@ div.problem {
     &:first-child {
       margin-bottom: $baseline / 4;
     }
+    &:active,
```

Comment:
I don't think we need this addition anymore, since we fixed the button styles this morning. My apology for the confusion. You can remove it before squashing. It doesn't hurt to leave it in there. Just to clean up.